doc/source/whatsnew/v0.24.0.txt

```diff
@@ -20,7 +20,8 @@ New features
 the user to override the engine's default behavior to include or omit the
 dataframe's indexes from the resulting Parquet file. (:issue:`20768`)
 - :meth:`DataFrame.corr` and :meth:`Series.corr` now accept a callable for generic calculation methods of correlation, e.g. histogram intersection (:issue:`22684`)
-
+- :func:`DataFrame.to_string` now accepts ``decimal`` as an argument, allowing
+the user to use a decimal separator. (:issue:`23614`)
```

Comment:
more than `to use` as decimal separator, if I understood correctly, it'll allow the user to specify which decimal separator should be used in the output. Isn't it?